app/services/cache_service.rb

```diff
@@ -37,11 +32,13 @@ def cache_only_updated_listings
         l['Id'] == fresh_listing['Id']
       end
 
-      cache_single_listing(fresh_listing) unless listing_unchanged?(prev_cached_listing,
-                                                                    fresh_listing) && listing_images_unchanged?(
-                                                                      prev_cached_listing, fresh_listing
-                                                                    )
-      cache_listing_images && process_listing_images(fresh_listing)
+      ## todo: remove listing_unchanged because it doesn't work?
```

Comment:
@cade-exygy I found that listing_unchanged _does_ work - do you want to pair and compare what we found?